### PR TITLE
OSDOCS-4199: Added support statement about multiple vCenter deployments

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.6+

Issue:
This PR addresses [OSDOCS-4199](https://issues.redhat.com//browse/OSDOCS-4199), which was opened per this Slack [thread](https://coreos.slack.com/archives/CBQHQFU0N/p1661771325587699).

Link to docs preview:

- [Installing a cluster on vSphere](https://51063--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
- [Installing a cluster on VMC](https://51063--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc.html)

Worth mentioning that the support note available in the preview has been added to all of the vSphere and VMC installation topics.

QE review:
Pending.

### CODE REVIEW

**Comment by mjpytlak** [comment]
@gnufied Ready for QE review. PTAL. Thank you.

**Comment by mjpytlak** [comment]
@jcpowermac and @jinyunma I added a note to all of our vSphere installation topics explicitly stating that installing a cluster to multiple vCenters is not supported. PTAL when you have a moment.

**Comment by jinyunma** [comment]
I checked that the update is only reflected in all docs under "Installing on vSphere", but missed in docs under "Installing on VMC".

**Comment by mjpytlak** [comment]
Thank you @jinyunma. Added the content to all of the VMC installation topics.

File: snippets/vcenter-support.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+:_content-type: SNIPPET
+
+[NOTE]
+====
+{product-title} supports deploying a cluster to a single VMware vCenter only. Deploying a cluster with machines/machine sets on multiple vCenters is not supported.
```
Comment (adellape) [inline snippets/vcenter-support.adoc:22]: Could this be "machines or machine sets" instead of the slash?

**Comment by jinyunma** [comment]
@mjpytlak thanks for the update.  LGTM now.
cc: @sgaoshang

File: snippets/vcenter-support.adoc (hunk 1 above)
Comment (mjpytlak) [reply snippets/vcenter-support.adoc:22]: Yes it can. Thank you.

Comment (mjpytlak) [reply snippets/vcenter-support.adoc:22]: I can going to leave this, as is.

**Pull request merged**